lib/Doctrine/DBAL/Driver/SQLAnywhere/SQLAnywhereStatement.php

```diff
@@ -110,6 +119,8 @@ public function bindParam($column, &$variable, $type = null, $length = null)
                 throw new SQLAnywhereException('Unknown type: ' . $type);
         }
 
+        $this->boundValues[$column] =& $variable;
```

Comment:
This should probably be `= &$variable`, but we have no CS rule for this afaik so it's a minor.

Reply:
done. But I made copy&paste from oci8-Statement ;-)